bricksrc/alarm.py

```diff
@@ -192,22 +192,19 @@
                         "tags": [TAG.Point, TAG.Water, TAG.Temperature, TAG.Alarm],
                         "parents": [BRICK.Water_Alarm],
                         "subclasses": {
-                            "Discharge_Water_Temperature_Alarm": {
```

Comment:
I think the idea was to keep deprecated concepts in the hierarchy. Is there a different way to preserve that even if we remove them here?

Reply:
They will be in the deprecation file